### ISSUE TITLE
Failed to Activate Extension in VScode for IBM BlockChain Platform 1.0.28

### ISSUE THREAD

**Issue by asifwaquar** [IBM-Blockchain/blockchain-vscode-extension#2359]
## Description
I am trying to install IBM Blockchain Extension 1.0.28 . 

## Expected Behavior
Extension should be activated

## Actual Behavior
<img width="457" alt="Screenshot 2020-05-24 at 1 45 30 PM" src="https://user-images.githubusercontent.com/5023738/82761067-5e87e200-9e2a-11ea-9f6c-89da751a832c.png">

I'm facing same issue not able to activate extension ,please advise.
my current versions:
node -v
v12.16.3
npm -v
6.14.4

Operating System Mac
<img width="457" alt="Screenshot 2020-05-25 at 1 54 27 AM" src="https://user-images.githubusercontent.com/5023738/82761104-b292c680-9e2a-11ea-9ef5-a2eb4f01a3a2.png">
 VS Code Version: 1.45.1
Electron: 7.2.4
Chrome: 78.0.3904.130
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0

## Steps to Reproduce

1.
2.
3.

## Existing issues

- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

## Your Environment

* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

**Comment by Jakeeyturner** [comment]
Hi @asifwaquar,

I suspect your problem is being caused by Node 12.
The extension only supports Node 8 & 10 - so you'll need to downgrade in order for the extension to work.

Thanks,
Jake

**Comment by asifwaquar** [comment]
thanks @Jakeeyturner , I have downgraded to node version 10 but still extension was not working with vs code version 1.45.1 .If directly install extension from vscode code .
But if we downgrade vscode to v1.39 it will work with node10.

However after installing extension manually using ibm-blockchain-platform-1.0.30.vsix file from github it is working with vs code version 1.45.1 also. I managed to install . It's working now.

thanks you so much .
Asif
